interactive_demo.py

```diff
@@ -0,0 +1,200 @@
+
+        # Input: List of joint positions matching order known by plant
+        self._joint_positions_port = self.DeclareVectorInputPort(
+            'joint_positions', BasicVector_[float](num_positions))
+        # Output: Rigid transforms of each body in base frame
+        self.DeclareAbstractOutputPort(
+            'transforms',
+            lambda: AbstractValue.Make([RigidTransform()]),
+            self._do_forward_kinematics)
+
+    def _do_forward_kinematics(self, context, data):
+        joint_positions = self._joint_positions_port.Eval(context)
+
+        # Set the latest positions in the plant context
+        self._plant.SetPositions(self._plant_context, joint_positions)
+
+        world_frame = self._plant.world_frame()
+
+        transforms = []
+        for i in range(self._plant.num_bodies()):
+            body = self._plant.get_body(BodyIndex(i))
+
+            # calculate pose of body in world frame
+            body_frame = body.body_frame()
+            transforms.append(
+                self._plant.CalcRelativeTransform(
+                    self._plant_context, world_frame, body_frame))
+
+        data.set_value(transforms)
+
+
+class PadVector(LeafSystem):
+    """
+    Add some constant values to pad a vector.
+    """
+
+    def __init__(self, num_inputs, num_outputs, output_value=0.0):
+        super().__init__()
+
+        if num_inputs > num_outputs:
+            raise ValueError('Must have more outputs than inputs')
+
+        self._difference = num_outputs - num_inputs
+        self._output_value = output_value
+
+        self._input_port = self.DeclareVectorInputPort(
+            'input', BasicVector_[float](num_inputs))
+
+        self.DeclareVectorOutputPort(
+            'output', BasicVector_[float](num_outputs), self._pad_output)
+
+    def _pad_output(self, context, output):
+        in_vec = self._input_port.Eval(context)
+        vector = [v for v in in_vec]
+        for i in range(self._difference):
+            vector.append(self._output_value)
+        output.SetFromVector(vector)
+
+
+def main():
+    builder = DiagramBuilder()
+
+    station = builder.AddSystem(ManipulationStation())
+    station.SetupClutterClearingStation()
+    station.Finalize()
+
+    # Create robot description with re-written paths
+    this_dir = os.path.abspath(os.path.dirname(__file__))
+    sdf_file_path = os.path.join(this_dir, 'iiwa14_no_collision.sdf')
+    with open(os.path.join(this_dir, 'iiwa14_no_collision.sdf.in'), 'r') as file_in:
+        with open(sdf_file_path, 'w') as file_out:
+            file_out.write(file_in.read().replace('PWD_GOES_HERE', this_dir))
+
+    joint_names = [
+        'iiwa_joint_1',
+        'iiwa_joint_2',
+        'iiwa_joint_3',
+        'iiwa_joint_4',
+        'iiwa_joint_5',
+        'iiwa_joint_6',
+        'iiwa_joint_7']
+    joints = []
+    for name in joint_names:
+        joints.append(station.get_controller_plant().GetJointByName(name))
```

Comment:
I'm getting the following error here:

```
Traceback (most recent call last):
  File "./interactive_demo.py", line 200, in <module>
    main()
  File "./interactive_demo.py", line 122, in main
    joints.append(station.get_controller_plant().GetJointByName(name))
TypeError: Unable to convert function return value to a Python type! The signature was
        (self: pydrake.examples.manipulation_station.ManipulationStation) -> drake::multibody::MultibodyPlant<double>
```

Any chance you know why?

Reply:
Found out why. I think `from pydrake.multibody.plant import MultibodyPlant` is needed in the imports to bring in the `drake::multibody::MultibodyPlant<double>` symbol

Reply:
Seems like a bug in the python bindings for manipulation station. Worked around it by importing ManipulationStation in 29464598cf94a29c4efd386351b62522d560613a